### PR TITLE
ankitRai96 and brainFuck

### PR DESCRIPTION
**Pull request by ankitrai96**
First of all, Kudos to all the efforts!

@AliceWonderland your repository is really awesome. And to be honest, people needed it. Beginners and Veterans alike. Felt good to revisit the basics again!

### CODE REVIEW

File: profiles/Ankit_Rai.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#### Get-in-Touch
+- web : [Ankit Rai](http://ankitrai.xyz)
+- mail: [abc@ankitrai.xyz](mailto:abc@ankitrai.xyz)
+
+#### Projects
+- [Power-Pi](https://github.com/ankitrai96/power-pi)
+- [multilingual Algorithm Vault](https://github.com/ankitrai96/multilingual-algorithm-vault)
+- [more on my GitHub ;)](https://github.com/ankitrai96/)
+
+#### Personal Details
```
Comment (StevenCopeland) [inline profiles/Ankit_Rai.md]: @ankitrai96 Are you sure you want to share all these personal details?

Comment (StevenCopeland) [reply profiles/Ankit_Rai.md]: If you are fine with it then I will merge to master.

Comment (ankitrai96) [reply profiles/Ankit_Rai.md]: Yeah, I'm okay with it @stevencopelan.

It just came to me at that moment. That's why I put it there. However, there's nothing sensitive or confidential about me in it.

Anyway, thanks. You can go ahead and merge.

Comment (StevenCopeland) [reply profiles/Ankit_Rai.md]: :+1:

**Pull request merged**